Brewfile

```diff
@@ -0,0 +1,2 @@
+brew "postgresql@10", start_service: true
+brew "redis", start_service: true
```

Comment:
This is just a little bit of setup help here, I'll wire it up into bin/setup in a new PR.